src/inverseRobot.js

```diff
@@ -24,7 +24,31 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const invRobot = {};
+
+  if (Object.keys(robot).length === 0) {
+    return invRobot;
+  }
+
+  for (const el in robot) {
+    let count = 0;
+
+    for (const secEl in robot) {
+      if (robot[el] === robot[secEl]) {
+        count++;
+      }
+    }
+
+    if (count > 1) {
+      return null;
+    }
+  }
+
+  for (const el in robot) {
+    invRobot[robot[el]] = el;
+  }
```

Comment:
you can check if key is already present in inversed robot here, go to [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) to find out how you can do it